cBinder/WrapperBuilder.py

```diff
@@ -50,7 +50,7 @@ def build_wrapper_for_structs_and_functions(self, header_name, enums, structs, f
                 writer.write_line('from cffi import FFI')
                 writer.write_line('ffi = FFI()')
 
-                enum_decls = writer.escaped('\n'.join(decl.declaration_string for decl in enums))
```

Comment:
may have actually exposed a bug here... enum declarations should also be included just like struct and function declarations.